cads_adaptors/mapping.py

```diff
@@ -2,6 +2,21 @@
 
 import copy
 
+DATE_KEYWORD_CONFIGS = [
+    {
+        "date_keyword": "date",
+        "year_keyword": "year",
+        "month_keyword": "month",
+        "day_keyword": "day",
+    },
+    {
+        "date_keyword": "hdate",
+        "year_keyword": "hyear",
+        "month_keyword": "hmonth",
+        "day_keyword": "hday",
+    },
+]
+
```

Comment:
This structure is a bit confusing to me - at first glance it wasn't clear that the year, month and day keywords are essentially the ctructural components of each date keyword. This my just be my brain though - but this is what my brain wants:
```suggestion
DATE_KEYWORD_CONFIGS = {
    "date": {
        "year": "year",
        "month": "month",
        "day": "day",
    },
    "hdate": {
        "year": "hyear",
        "month": "hmonth",
        "day": "hday",
    },
}
```

Reply:
As discussed over the desk-divider, this was as good as (if not better than) any of the other options, so we will keep it as it is.